INSTALL.rst

```diff
@@ -304,13 +304,31 @@ You should also set apache to start on boot::
 
   sudo chkconfig httpd on
 
-The networking server may be started by running::
+Running the network server:
+---------------------------
 
-  haas serve_networks &
 
-as the HaaS user. To make this happen on boot, add the following to ``/etc/rc.local``::
+A systemd script for running the network server is available in the 'scripts' directory.
+Name of the script is: haas_network.service
+
+Centos uses systemd to controll all its processes. 
+Ubuntu does not use systemd by default. To use the following script
+on ubuntu server, you will have to install systemd.
```

Comment:
Newer versions of ubuntu [come with systemd by default](https://wiki.ubuntu.com/SystemdForUpstartUsers).